### PR TITLE
Update connect-a-publishing-site-to-a-catalog.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed the invalid link with https://docs.microsoft.com/en-us/sharepoint/create-and-manage-terms

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/administration/connect-a-publishing-site-to-a-catalog.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ Before you connect the publishing site collection to a catalog, review the infor
 If you want to remove the content of a connected catalog from a publishing site, you have to disconnect the publishing site from the catalog.
   
 > [!IMPORTANT]
-> If you have integrated a catalog tagging term set into the publishing site navigation term set, the tagging terms will not be removed from the navigation when you disconnect from the catalog. To remove the tagging terms from the navigation, you have to delete the terms in Term Store Management. For more information, see [Create and manage terms in a term set](https://office.microsoft.com/en-us/office365-sharepoint-online-enterprise-help/create-and-manage-terms-in-a-term-set-HA102771989.aspx)
+> If you have integrated a catalog tagging term set into the publishing site navigation term set, the tagging terms will not be removed from the navigation when you disconnect from the catalog. To remove the tagging terms from the navigation, you have to delete the terms in Term Store Management. For more information, see [Create and manage terms in a term set](https://docs.microsoft.com/en-us/sharepoint/create-and-manage-terms)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/connect-a-publishing-site-to-a-catalog.md]:
```suggestion
> If you have integrated a catalog tagging term set into the publishing site navigation term set, the tagging terms will not be removed from the navigation when you disconnect from the catalog. To remove the tagging terms from the navigation, you have to delete the terms in Term Store Management. For more information, see [Create and manage terms in a term set](https://docs.microsoft.com/en-us/sharepoint/create-and-manage-terms).
```

File: SharePoint/SharePointServer/administration/connect-a-publishing-site-to-a-catalog.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ Before you connect the publishing site collection to a catalog, review the infor
 If you want to remove the content of a connected catalog from a publishing site, you have to disconnect the publishing site from the catalog.
   
 > [!IMPORTANT]
-> If you have integrated a catalog tagging term set into the publishing site navigation term set, the tagging terms will not be removed from the navigation when you disconnect from the catalog. To remove the tagging terms from the navigation, you have to delete the terms in Term Store Management. For more information, see [Create and manage terms in a term set](https://office.microsoft.com/en-us/office365-sharepoint-online-enterprise-help/create-and-manage-terms-in-a-term-set-HA102771989.aspx)
+> If you have integrated a catalog tagging term set into the publishing site navigation term set, the tagging terms will not be removed from the navigation when you disconnect from the catalog. To remove the tagging terms from the navigation, you have to delete the terms in Term Store Management. For more information, see [Create and manage terms in a term set](https://docs.microsoft.com/en-us/sharepoint/create-and-manage-terms).
```
Comment (nenonix) [inline SharePoint/SharePointServer/administration/connect-a-publishing-site-to-a-catalog.md]:
```suggestion
> If you have integrated a catalog tagging term set into the publishing site navigation term set, the tagging terms will not be removed from the navigation when you disconnect from the catalog. To remove the tagging terms from the navigation, you have to delete the terms in Term Store Management. For more information, see [Create and manage terms in a term set](https://docs.microsoft.com/sharepoint/create-and-manage-terms).
```

**Comment by scanum** [comment]
@JoanneHendrickson Changes look correct. Could you please review and merge? Thanks!

**Pull request merged**